css/style.scss

```diff
@@ -60,13 +60,21 @@ html {
 .layer-two {
     width: 100%;
     height: 100%;
+<<<<<<< HEAD
     background: linear-gradient(45deg, #73e343 35%, #E34374 35% 73.5%, #F4F008 73.5%);
+=======
+    background: linear-gradient(45deg, #2845AD 35%, #E34374 35% 75%, #F4F008 75%);
+>>>>>>> 1db078439423f82da4a17ec0acee578fb455964c
 }
 
 .layer-three {
     width: 100%;
     height: 100%;
+<<<<<<< HEAD
     background: linear-gradient(135deg, rgba(255, 255, 255, 0) 65%, #43a6e3 65%);
+=======
+    background: linear-gradient(135deg, rgba(255, 255, 255, 0) 65%, #8756C7 65%);
+>>>>>>> 1db078439423f82da4a17ec0acee578fb455964c
```

Comment:
@liuba-larousse, when you do a pull from another branch, sometimes there are conflicts, you should solve them before pushing the changes. [Here's a short tutorial about it](https://www.youtube.com/watch?v=lXPYqVjyqHM)